doc/requirements-rtd.txt

```diff
@@ -1,13 +0,0 @@
-Pygments==2.3.1
```

Comment:
The reason why this is here / in the instructions is because it allows us to reproduce the sphinx build on readthedocs exactly. Is there a reason to remove?

Reply:
accidental, thanks for spotting this.